lambda/lambda-metering-users.py

```diff
@@ -0,0 +1,52 @@
+import base64
+import gzip
+import json
+import os
+from botocore.exceptions import ClientError
+import boto3
+import logging
+from datetime import datetime
+
+def lambda_handler(event, context):
+    cw_data = event['awslogs']['data']
+    compressed_payload = base64.b64decode(cw_data)
+    uncompressed_payload = gzip.decompress(compressed_payload)
+    data = json.loads(uncompressed_payload.decode('utf-8'))
+    sqs_url = os.environ.get("SqsMeteringUsersUrl")
+    customerID = os.environ.get("customerID")
+    logging.getLogger().setLevel(logging.ERROR)
+    users = 0
+    sqs = boto3.client('sqs')
+
+    for event in data['logEvents']:
+        message = json.loads(event['message'])
+        source = message['eventSource']
+        event = message['eventName']
+        
+        if event == 'DeleteUser':
+            users = -1
+        elif event == 'CreateUser':
+            users = 1
+        
+        msm_body = { 
+            "Type": "Notification", 
+            "Message" : {
+                "action" : "users-updated",
+                "customer_id": customerID,
+                "user_numbers": users,
+                "time": str(datetime.now())
```

Comment:
Change `str(datetime.now())` to `datetime.now().isoformat(" ")`.

`str` is intended for debugging purpose (so its output is not guaranteed to stay stable), while `.isoformat()` guarantees the output format of the timestamp.